From 5c8fa2301ee9fa92267ff351e3fa3e59f2b2df79 Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Wed, 24 Jan 2018 16:01:48 -0600
Subject: [PATCH] Partially revert upstream commit
 f7c3a4a637d59f3065d8129e9ebacba992dfc469

Upstream converted the code based on rabbitmqctl 3.6.10 changes
We are using 3.6.5  but hopefully this expression will match both

Status of node rabbit@localhost ...
or
Status of node rabbit@localhost
---
 lib/facter/rabbitmq_nodename.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/facter/rabbitmq_nodename.rb b/lib/facter/rabbitmq_nodename.rb
index 301e3c1..d5c6352 100644
--- a/lib/facter/rabbitmq_nodename.rb
+++ b/lib/facter/rabbitmq_nodename.rb
@@ -2,7 +2,7 @@ Facter.add(:rabbitmq_nodename) do
   setcode do
     if Facter::Core::Execution.which('rabbitmqctl')
       rabbitmq_nodename = Facter::Core::Execution.execute('rabbitmqctl status 2>&1')
-      %r{^Status of node '?([\w\.]+@[\w\.\-]+)'?}.match(rabbitmq_nodename)[1]
+      %r{^Status of node '?([\w\.]+@[\w\.\-]+)'?( \.+)?$}.match(rabbitmq_nodename)[1]
     end
   end
 end
-- 
1.8.3.1

